src/ray/core_worker/lib/java/io_ray_runtime_object_NativeObjectStore.cc

```diff
@@ -35,7 +40,8 @@ ray::Status PutSerializedObject(JNIEnv *env, jobject obj, ray::ObjectID object_i
   if (object_id.IsNil()) {
     status = ray::CoreWorkerProcess::GetCoreWorker().CreateOwned(
         native_ray_object->GetMetadata(), data_size, native_ray_object->GetNestedIds(),
-        out_object_id, &data, /*created_by_worker=*/true);
+        out_object_id, &data, /*created_by_worker=*/true,
+        /*owner_address=*/std::move(owner_address));
```

Comment:
It looks like you continue to use a lot of `owner_address`, It looks very dangerous to use `move` in here.

Hmm, it seems my mistake: maybe use '&owner_address' in [CreateOwned](https://github.com/ray-project/ray/blob/01158d679c47ac5a07a2314d71ed9defa93eae5a/src/ray/core_worker/core_worker.h#L547) can solve it.

Reply:
+1 for not moving owner_address.

Reply:
Okay, shall I include the change in this PR?

Reply:
I've updated the `CreateOwned` function. Is this OK?

Reply:
I think it's great!